.github/pull_request_template.md

```diff
@@ -14,3 +14,21 @@
 - complaints about how bad the code is
 
 ## Screenshots / Demos
+
+## Checklist for Author and Reviewer
+
+### Testing
+- [ ] Includes a summary of what a code reviewer should verify
+- [ ] Changes are well-tested and code coverage remains the same or increases
+
+### Breaking Changes
+- [ ] This PR does not introduce any breaking changes
```

Comment:
```suggestion
```

I would remove this. I think the next two check boxes effectively get at the know issues